src/Receipt/DonationDetailsGroup/Details/Status.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Give\Receipt\DonationDetailsGroup\Details;
+
+use Give\Receipt\Detail;
+
+/**
+ * Class Status
+ *
+ * @since 2.7.0
+ * @package Give\Receipt\DonationDetailsGroup\Details
+ */
+class Status extends Detail {
+	/**
+	 * @inheritDoc
+	 */
+	public function getLabel() {
+		return __( 'PAYMENT STATUS', 'give' );
```

Comment:
Because translated strings are user-defined, the translated string should be escaped.